sequencing-server/test/seqjson-to-edsl.spec.ts

```diff
@@ -250,13 +250,11 @@ describe('getEdslForSeqJson', () => {
       }
     ],
     hardware_commands: [
-      {
-        description: 'FIRE THE PYROS',
-        metadata:{
-          author: 'rrgoetz',
-        },
-        stem: 'HDW_PYRO_ENGINE',
-      }
+      HDW_PYRO_ENGINE
```

Comment:
Same as my other comment about this command.

Reply:
You can add a fake stem here because we are not using the command dictionary to go back to eDSL. This will be similar if the user uploads a random SeqJson that doesn't match an uploaded command dictionary.